thundra/utils.py

```diff
@@ -304,5 +304,6 @@ def parse_http_url(url, url_path_depth):
 EXCLUDE_EXCEPTION_URLS = {
     str.__contains__: [
         'execute-api',
+        'elb'
```

Comment:
I think we can use `.elb.` instead of `elb`, since it is very short and possibly included in a URL.